service/keys/keys.go

```diff
@@ -41,6 +48,36 @@ func (keys *keys) Init(cfg config.Config, encryptionKey string) error {
 	}
 	keys.nostrSecretKey = nostrSecretKey
 	keys.nostrPublicKey = nostrPublicKey
+
+	mnemonic, err := cfg.Get("Mnemonic", encryptionKey)
+	if err != nil {
+		logger.Logger.WithError(err).Error("Failed to decrypt mnemonic")
+		return err
+	}
+
+	if mnemonic != "" {
+		masterKey, err := bip32.NewMasterKey(bip39.NewSeed(mnemonic, ""))
+		if err != nil {
+			logger.Logger.WithError(err).Error("Failed to create seed from mnemonic")
+			return err
+		}
+
+		APP_INDEX := uint32(0) // TODO: choose an index
+		appKey, err := masterKey.NewChildKey(APP_INDEX)
+		if err != nil {
+			logger.Logger.WithError(err).Error("Failed to create seed from mnemonic")
+			return err
+		}
+
+		ENCRYPTED_SCB_INDEX := uint32(0) // TODO: choose an index
+		encryptedChannelsBackupKey, err := appKey.NewChildKey(ENCRYPTED_SCB_INDEX)
+		if err != nil {
+			logger.Logger.WithError(err).Error("Failed to create seed from mnemonic")
+			return err
+		}
+		keys.encryptedChannelsBackupKey = hex.EncodeToString(encryptedChannelsBackupKey.Key)
```

Comment:
do we need to store those on startup or is it enough to store the master key and then derive keys on the fly where ever it is needed. 

I am wondering if this `keys` part of the application needs to know about the `backup` part. 
ideally it's all a small piece that functions independently and knows as little as possible from other parts I think. With this it is universal/abstract and can be reused.